src/schema/v2/SaleAgreements/SaleAgreement.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLFieldConfig,
+  GraphQLID,
+  GraphQLBoolean,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import { markdown } from "../fields/markdown"
```

Comment:
#minor i think mild pref. for absolute paths in imports (makes for easier copy/paste)